scripts/lib/install

```diff
@@ -222,7 +222,7 @@ case "$os_id $os_version_id" in
     'debian 11' | 'debian 12' | 'ubuntu 20.04' | 'ubuntu 22.04') ;;
     *)
         set +x
-        cat <<EOF
+        cat <<EOF >&2
```

Comment:
I checked that `scripts/setup/install` redirects stderr to stdout; so does this accomplish anything?

Reply:
I guess not. We have other error messages that go to stderr (before the redirection), so I wanted to be consistent, but we can worry about that separately or never.